numpy/typing/tests/data/reveal/ma.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+import numpy as np
+
+from typing_extensions import assert_type
+
+arr: np.ma.MaskedArray[Any, np.dtype[np.float64]] = np.ma.masked_array([[1.0, 2.0]], mask=[True, False])
+result = np.ma.min(arr, axis=None)
+assert_type(result, np.float64)
+
+tgt: np.ma.MaskedArray[Any, np.dtype[np.float64]] = np.ma.empty(shape=())
+result = np.ma.min(arr, axis=None, out=tgt)
+assert_type(result, np.ma.MaskedArray)
```

Comment:
The type of `tgt` is equivalent to that of `arr`, so there's no need for it (so it doesn't matter that at runtime this wouldn't make a lot of sense).

Additionally, `np.ma.MaskedArray` should have type arguments, as its type parameters have no defaults (PEP 696).

```suggestion
assert_type(np.ma.min(arr, axis=None, out=arr), np.ma.MaskedArray[Any, np.dtype[np.float64]])
```